packages/material-ui/src/GridListTileBar/GridListTileBar.js

```diff
@@ -72,33 +72,35 @@ const GridListTileBar = React.forwardRef(function GridListTileBar(props, ref) {
     actionIcon,
     actionPosition = 'right',
     classes,
-    className: classNameProp,
+    className,
     subtitle,
     title,
     titlePosition = 'bottom',
     ...other
   } = props;
 
   const actionPos = actionIcon && actionPosition;
```

Comment:
If you inline this then `optimize-clsx` can produce smaller code

```javascript
var actionPos = actionIcon && actionPosition;

clsx(classes.titleWrap, {
  left: classes.titleWrapActionPosLeft,
  right: classes.titleWrapActionPosRight
}[actionPos]);
```
vs
```
clsx(classes.titleWrap, actionIcon && (actionPosition === 'left' ? classes.titleWrapActionPosLeft : classes.titleWrapActionPosRight));
```

Reply:
Thanks for the info. I think that we can revisit later.